solutionbox/inception/datalab_solutions/inception/_package.py

```diff
@@ -142,44 +142,49 @@ def cloud_train(input_dir, batch_size, max_steps, output_dir,
 def _display_predict_results(results, show_image):
   if (_util.is_in_IPython()):
     import IPython
-    for image_file, label_and_score in results:
+    for image, label_and_score in results:
       if show_image is True:
         IPython.display.display_html('<p style="font-size:28px">%s(%.5f)</p>' % label_and_score,
             raw=True)
-        with ml.util._file.open_local_or_gcs(image_file, mode='r') as f:
-          IPython.display.display(IPython.display.Image(data=f.read()))
+        IPython.display.display(IPython.display.Image(data=image))
       else:
         IPython.display.display_html(
             '<p>%s&nbsp&nbsp%s(%.5f)</p>' % ((image_file,) + label_and_score), raw=True)
   else:
     print results
 
 
-def local_predict(model_dir, image_files, show_image=True):
+def local_predict(model_dir, image_files, resize=False, show_image=True):
   """Predict using an offline model.
   Args:
     model_dir: The directory of a trained inception model. Can be local or GCS paths.
     image_files: The paths to the image files to predict labels. Can be local or GCS paths.
     show_image: Whether to show images in the results.
+    resize: Whether to resize the image to a reasonable size before prediction.
```

Comment:
what is reasonable, can I set set the size. What happens if the image is smaller than expected? Just add these points to the comment.